log.js

```diff
@@ -23,8 +23,8 @@ var logger = new winston.Logger({
   exitOnError: false
 });
 
-if (!fs.existsSync('./logs/')) {
-  fs.mkdirSync('logs')
+if (!fs.existsSync('./logs/')) {  // eslint-disable-line no-sync
+  fs.mkdirSync('logs');   // eslint-disable-line no-sync
```

Comment:
With these being relative paths, rather than absolute, is there any chance that they could end up in inconsistent eventual paths? I was wondering the same about the references in `server.js` to `'./.kibana_items.json'` and `'./.es_template.js'`. I also see that you are using `path.resolve(...)` for finding the config file. Would this be a safer way to handle these path references?

Reply:
I considered letting the user specify the log path but that would be another setting/parameter ... i feel like I can't just do `/var/log` because that's not cross-platform compatible ... any advise on where best to store logs? If linux -> `/var/log/`, if Windows -> ???
I'm using path.resolve() for the config file which is provided as a command line parameter so i don't know what the path looks like.
The two files you mention are distributed as part of the code/repo, not by the user. They should be in the expected path.

All tips/advise on this are very welcome!